webapp/src/test/java/io/github/microcks/util/URIBuilderTest.java

```diff
@@ -127,4 +129,27 @@ public void testBuildURIFromPatternWithMap() {
       assertTrue("http://localhost:8080/blog/2018/05?page=0&status=published".equals(uri)
             || "http://localhost:8080/blog/2018/05?status=published&page=0".equals(uri));
    }
+
+  @Test
+  public void testBuildURIFromPatternWithMapWithParamsArray() {
+    // Prepare a bunch of parameters.
+    Multimap<String, String> parameters = ArrayListMultimap.create();
+    parameters.put("year", "2018");
+    parameters.put("month", "05");
+    parameters.put("status", "published");
+    parameters.put("status", "proofred");
+    parameters.put("page", "0");
+
+    // Test with old wadl like template format.
+    String pattern = "http://localhost:8080/blog/{year}/{month}";
+    String uri = URIBuilder.buildURIFromPattern(pattern, parameters);
+    assertTrue("http://localhost:8080/blog/2018/05?page=0&status=published".equals(uri)
```

Comment:
Shouldn't we have both status (`proofred` and `published`) present in the URI here?

Reply:
They should :D will fix that